app/models.py

```diff
@@ -238,6 +220,11 @@ class BlockBehaviourEnum(EnumE):
     return_5xx = 1
 
 
+class UnsubscribeBehaviourEnum(EnumE):
```

Comment:
This should rather be in the next PR right?

Reply:
:+1: